dotCMS/src/integration-test/java/com/dotcms/scheduler/TestSynchronizedTask.java

```diff
@@ -0,0 +1,36 @@
+package com.dotcms.scheduler;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import io.vavr.control.Try;
+
+public class TestSynchronizedTask extends DotSyncronizedTask {
+
+    private final static String SYSTEM_INCREMENT_KEY = "TestSynchronizedTaskKey";
+    private final static String TASK_RUNNING_KEY = "TestSynchronizedTaskRunning";
+    public static int getTotalIncrement() {
+        return Try.of(() -> Integer.parseInt(System.getProperty(SYSTEM_INCREMENT_KEY))).getOrElse(0);
+    }
+    
+    final int myIncrement;
+    
+    public TestSynchronizedTask(int inc) {
+        this.myIncrement=inc;
+    }
+
+    @Override
+    public void execute() {
+        if (System.getProperty(TASK_RUNNING_KEY) != null) {
+            throw new DotRuntimeException("Another task is already running");
+        }
+        try {
+            System.setProperty(TASK_RUNNING_KEY, "true");
+            System.out.println("TestSynchronizedTask start:" + getTotalIncrement() + " adding:" + myIncrement );
```

Comment:
Codacy found an issue: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)